requirements/requirements.txt

```
@@ -2,6 +2,7 @@ colorama
 distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
 flashbax~=0.1.0
 flax~=0.7.5
+gigastep @ git+https://github.com/mlech26l/gigastep
```

We can just use their pypi package

```suggestion
gigastep
```

---

removing the  `@ git+https://github.com/mlech26l/gigastep ` results in an empty gigastep folder (ie you cant use any functions) for whatever reason so  having it there is required